docs/core/introduction.md

```diff
@@ -1,238 +1,84 @@
-
-For more information, see the following resources:
-
-* [.NET In-Browser Tutorial](https://dotnet.microsoft.com/learn/dotnet/in-browser-tutorial/1)
-* [Using .NET notebooks with Jupyter on your machine](https://github.com/dotnet/interactive/blob/main/docs/NotebookswithJupyter.md)
-* [.NET Interactive documentation](https://github.com/dotnet/interactive/blob/main/docs/README.md)
-
-## CI/CD
-
-MSBuild and the .NET CLI can be used with various continuous integration tools and environments, such as:
-
-* [GitHub Actions](https://github.com/features/actions)
-* [GitHub Actions and .NET](../devops/github-actions-overview.md)
-* [Azure DevOps](/azure/devops/user-guide/what-is-azure-devops)
-* [CAKE for C#](https://cakebuild.net/)
-* [FAKE for F#](https://fake.build/)
-
-For more information, see [Use the .NET SDK in Continuous Integration (CI) environments](../devops/dotnet-cli-and-continuous-integration.md).
-
-## Deployment models
-
-.NET apps can be [published in two different modes](deploying/index.md):
-
-* *Self-contained* apps include the .NET runtime and dependent libraries. They can be [single-file](deploying/single-file/overview.md) or multi-file. Users of the application can run it on a machine that doesn't have the .NET runtime installed. Self-contained apps always target a single operating system and architecture configuration.
-* *Framework-dependent* apps require a compatible version of the .NET runtime, typically installed globally. Framework-dependent apps can be published for a single operating system and architecture configuration or as "portable," targeting all supported configurations.
-
-.NET apps are launched with a native executable, by default. The executable is both operating-system and architecture-specific. Apps can also be launched with the [`dotnet` command](tools/dotnet.md).
-
-Apps can be [deployed in containers](docker/introduction.md). Microsoft provides [container images](https://mcr.microsoft.com/catalog?search=dotnet) for various target environments.
-
-## .NET history
-
-In 2002, Microsoft released [.NET Framework](../framework/get-started/overview.md), a development platform for creating Windows apps. Today .NET Framework is at version 4.8 and remains [fully supported by Microsoft](https://dotnet.microsoft.com/platform/support/policy/dotnet-framework).
-
-In 2014, Microsoft introduced .NET Core as a cross-platform, open-source successor to .NET Framework. This new [implementation of .NET](../standard/glossary.md#implementation-of-net) kept the name .NET Core through version 3.1. The next version after .NET Core 3.1 dropped the "Core" part of the name and was named .NET 5.
+.NET is [supported by multiple organizations](https://github.com/dotnet/core/blob/main/support.md) that work to ensure .NET is supported on [multiple operating systems](https://github.com/dotnet/core/blob/main/os-lifecycle-policy.md), and kept up to date. It can be used on Arm64, x64, and x86 architectures.
```

Comment:
```suggestion
.NET is [supported by multiple organizations](https://github.com/dotnet/core/blob/main/support.md) that work to ensure that .NET can run on [multiple operating systems](https://github.com/dotnet/core/blob/main/os-lifecycle-policy.md) and kept up to date. It can be used on Arm64, x64, and x86 architectures.
```

Reply:
(Removes double "supported" in same sentence.)